terraform/dev/main.tf

```diff
@@ -17,7 +17,7 @@ terraform {
 }
 
 module "gke" {
-  source = "git::git@github.com:cloudkite-io/terraform-modules.git//modules/gke"
+  source = "git::git@github.com:cloudkite-io/terraform-modules.git//modules/gke?ref=v0.0.1"
```

Comment:
URL has an extra `/`.

Reply:
It requires that extra `/`

Reply:
https://www.terraform.io/docs/modules/sources.html#modules-in-package-sub-directories